Parsers/ASimNetworkSession/ProductParsers/NetworkSessionMicrosoftWindowsFirawall.yaml

```diff
@@ -0,0 +1,116 @@
+Parser:
```

Comment:
In addition to the below:
- You do need to construct EventResult and DvcAction from the data you have.
- EventEndTime and EventStartTime are mandatory. If no other info, set them to TimeGenerated.
- Add "EventSchema" and set it to "NetworkSessions". We discussed it yesterday or Thursday and I am going to add it to the schema doc.
- You need to handle Dvc and related fields similarly to other [WindowsEvent parsers](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimProcessEvent/ProductParsers/ProcessEventMicrosoftWindowsEventCreate.yaml) 
- Also, assume the source or dest (based on direction) hostname is Dvc.